packages/fhir-team-management/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Change Log
```

Comment:
Should this be here, its autogenerated by lerna after publishing the package, does not seem like this package is published yet.

Reply:
i am not sure .... ok let me remove this then if its something auto-generated